stores/in_memory/slashing_store.go

```diff
@@ -46,17 +45,17 @@ func (store *InMemStore) RetrieveProposal(key e2types.PublicKey, slot uint64) (*
 	return ret, nil
 }
 
-func (store *InMemStore) SaveLatestAttestation(key e2types.PublicKey, req *core.BeaconAttestation) error {
-	store.attMemory[hex.EncodeToString(key.Marshal())+"_latest"] = req
-	return nil
-}
-
-func (store *InMemStore) RetrieveLatestAttestation(key e2types.PublicKey) (*core.BeaconAttestation, error) {
-	return store.attMemory[hex.EncodeToString(key.Marshal())+"_latest"], nil
-}
+//func (store *InMemStore) SaveLatestAttestation(key e2types.PublicKey, req *core.BeaconAttestation) error {
+//	store.attMemory[hex.EncodeToString(key.Marshal())+"_latest"] = req
+//	return nil
+//}
+//
+//func (store *InMemStore) RetrieveLatestAttestation(key e2types.PublicKey) (*core.BeaconAttestation, error) {
+//	return store.attMemory[hex.EncodeToString(key.Marshal())+"_latest"], nil
+//}
 
-func attestationKey(key e2types.PublicKey, targetEpoch uint64) string {
-	return fmt.Sprintf("%s_%d", hex.EncodeToString(key.Marshal()), targetEpoch)
+func attestationKey(key e2types.PublicKey) string {
+	return fmt.Sprintf("%s", hex.EncodeToString(key.Marshal()))
```

Comment:
we don't need `fmt.Sprintf` here, just `return hex.EncodeToString(key.Marshal())`